### PR TITLE
Dep support

### LINKED ISSUE #5: Add mod or dep support?

### PR DESCRIPTION
**Pull request by ExFly**
closes #5

### CODE REVIEW

File: Gopkg.toml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+#   source = "github.com/myfork/project2"
+#
+# [[override]]
+#   name = "github.com/x/y"
+#   version = "2.4.0"
+#
+# [prune]
+#   non-go = false
+#   go-tests = true
+#   unused-packages = true
```
Comment (jonatasbaldin) [inline Gopkg.toml]: Hey @ExFly, thanks for this!

I think we can delete this commented lines 🤔 I was taking a look at other projects using `dep` and they don't have it.

File: README.md [hunk 2]
```diff
@@ -3,13 +3,16 @@ Download .gitignore files from the GitHub [gitignore](https://github.com/github/
 
 [![Build Status](https://travis-ci.org/jonatasbaldin/ignore.svg?branch=master)](https://travis-ci.org/jonatasbaldin/ignore)
 
-### Using it
+## Using it
+
```
Comment (jonatasbaldin) [inline README.md]: Can we remove this blank lines separating the headers and bodys? I do prefer they together 😊

File: README.md [hunk 3]
```diff
@@ -21,9 +24,22 @@ AppEngine
 ```
 
 Downloading a file:
+
 ```
 $ ignore Python
 ```
 
+## Contributing
```
Comment (jonatasbaldin) [inline README.md]: This is awesome, thanks :)

**Comment by jonatasbaldin** [comment]
Hey @ExFly, any update on this? ✨

**Comment by ExFly** [comment]
sorry, this rep is small, so I think the dep can be delay

**Comment by jonatasbaldin** [comment]
Why do you think that? Does it add too much complexity for a small repo?

**Pull request closed without merging**